docusaurus.config.js

```diff
@@ -28,13 +28,13 @@ const config = {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
           // Please change this to your repo.
-          editUrl: 'https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/',
+          editUrl: 'https://github.com/techconative/restel-docs/blob/main/',
```

Comment:
Link is not accessible. Can we just say https://github.com/techconative/restel-docs ?